src/screens/VerifyNumberScreen.tsx

```diff
@@ -48,6 +48,7 @@ function VerifyNumber({navigation, route}: Props) {
 
       return authActivate({passport_id, otp})
         .then(() => {
+          getPatient()
```

Comment:
This seems a bit odd to me, shouldn't this function be a promise, which then acts like:

```
getPatient().then(() => {navigation.replace(SCREENS.HOME)})
```

Reply:
It's just temporary - please ignore it for now